cpp/include/cudf/lists/lists_column_view.hpp

```diff
@@ -31,6 +31,7 @@ namespace cudf {
  */
 class lists_column_view : private column_view {
  public:
+  lists_column_view() = default;
```

Comment:
C++ reviewers: Cython likes classes with default constructors, so it'd be nice to have this unless it's explicitly bad for some reason.